awscli/customizations/wizard/ui/utils.py

```diff
@@ -31,6 +31,14 @@ def get_ui_control_by_buffer_name(layout, buffer_name):
         f"Couldn't find buffer in the current layout: {buffer_name}")
 
 
+def move_to_previous_prompt(traverser, layout):
```

Comment:
Since we have only one `traverser` and only one `layout` probably makes sense to get them from the app inside function instead of passing them in as arguments every time.

Reply:
I'm fine with making that change.